auto_submit/lib/service/github_service.dart

```diff
@@ -26,19 +26,21 @@ class GithubService {
     return await github.checks.checkRuns.listCheckRunsForRef(slug, ref: ref).toList();
   }
 
-  /// Retrieves the check suites with the ref.
-  Future<List<CheckSuite>> getCheckSuites(
-    RepositorySlug slug,
-    String ref,
-  ) async {
-    return await github.checks.checkSuites.listCheckSuitesForRef(slug, ref: ref).toList();
-  }
-
   /// Retrieves the statuses of a repository at the specified reference.
   Future<List<RepositoryStatus>> getStatuses(
     RepositorySlug slug,
     String ref,
   ) async {
     return await github.repositories.listStatuses(slug, ref).toList();
   }
+
+  /// Fetches the specified commit.
+  Future<RepositoryCommit> getRepoCommit(RepositorySlug slug, String sha) async {
+    return await github.repositories.getCommit(slug, sha);
+  }
+
+  /// Compares two commits
```

Comment:
Can you add some more context?

Reply:
Sure Casey, I added more context to explain what compareCommits() is doing and what its response means.